runtime/jcl/common/thread.cpp

```diff
@@ -617,6 +617,7 @@ jobject JNICALL
 Java_java_lang_Thread_findScopedValueBindings(JNIEnv *env, jclass unusedClass)
 {
 	/* TODO: Implement. See https://github.com/eclipse-openj9/openj9/issues/16677. */
```

Comment:
```suggestion
	/* Currently, this method is unused since there is no API or test that
         * invokes this method in OpenJ9. If the assertion ever triggers, the
         * draft implementation below should be completed and merged.
         *
         * Issue: https://github.com/eclipse-openj9/openj9/issues/16677.
         * Implementation: https://github.com/eclipse-openj9/openj9/pull/17402.
         */
```

Reply:
@thallium
- Also, employ the above wording in the commit message.
- Before closing  #17402, post a comment listing the issues which need to be resolved.